src/main/java/com/importer/fileimporter/utils/ProcessFileServiceUtils.java

```diff
@@ -0,0 +1,127 @@
+package com.importer.fileimporter.utils;
+
+import lombok.experimental.UtilityClass;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@UtilityClass
+public class ProcessFileServiceUtils {
+
+    public final String PAIR_KEY = "Pair";
+    public final String EXECUTED_KEY = "Executed";
+    public final String AMOUNT_KEY = "Amount";
+    public final String FEE_KEY = "Fee";
+    public final String PRICE_KEY = "Price";
+    public final String DATE_KEY = "Date(UTC)";
+    public final String SIDE_KEY = "Side";
```

Comment:
we should have a class mapping this keys to make it dynamically and be able to do it for multiple files with different names